CMakeLists.txt

```diff
@@ -3,10 +3,11 @@
 
 cmake_minimum_required(VERSION 3.25)
 
-project(maddy)
+project(maddy VERSION 1.5.0)
```

Comment:
Setting the `VERSION` is needed. Once there's a new release it should be updated, so consumers of the library can specify a required version (if needed).